setup.cfg

```
@@ -30,7 +30,7 @@ project_urls =
 packages = find:
 install_requires =
     attrs>=20.3,<22
-    click~=7.1
+    click>=7.1
```

```suggestion
    click>=7.1,<9
```

cheers, still rather mitigate future breaking changes though 😓